From 471c2ae7d472f8d9a83f3fa34d9555c5f423af16 Mon Sep 17 00:00:00 2001
From: Thomas Ackermann <th.acker66@arcor.de>
Date: Sat, 15 Dec 2012 14:56:37 +0100
Subject: [PATCH] Move api-command.txt to the end of API list in api-index.txt

- because it describes a different form of API than the other api-* documents

Signed-off-by: Thomas Ackermann <th.acker@arcor.de>
---
 Documentation/technical/api-index.sh | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/Documentation/technical/api-index.sh b/Documentation/technical/api-index.sh
index 9c3f413..c2c68ed 100755
--- a/Documentation/technical/api-index.sh
+++ b/Documentation/technical/api-index.sh
@@ -10,12 +10,16 @@
 	while read filename
 	do
 		case "$filename" in
-		api-index-skel.txt | api-index.txt) continue ;;
+		api-index-skel.txt | api-index.txt | api-command.txt) continue ;;
 		esac
 		title=$(sed -e 1q "$filename")
 		html=${filename%.txt}.html
 		echo "* link:$html[$title]"
 	done
+	filename=api-command.txt
+	title=$(sed -e 1q "$filename")
+	html=${filename%.txt}.html
+	echo "* link:$html[$title]"
 	echo "$c"
 	sed -n -e '/^\/\/ table of contents end/,$p' "$skel"
 ) >api-index.txt+
-- 
1.8.0.msysgit.0

